UI/Steps/Steps.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UI.Pages;
+
+namespace UI.Steps
+{
+    public static class Steps
+    {
+        private static LoginPage loginPage = new LoginPage();
+        private static MainPage mainPage = new MainPage();
+
+        public static void Login(string username, string password, int timeSeconds)
```

Comment:
Please create two files:
1. LoginPageSteps which will contain Login function
2. MainPageSteps which will contan Logout and Search functions